pennylane/fermi/fermionic.py

```diff
@@ -158,11 +160,15 @@ def __pow__(self, value):
 
         return operator
 
+    def to_qubit(self):
```

Comment:
We can add the same method for `FermiSentence` in this PR.

Reply:
done!